lua/cfc_adverts/server/sv_adverts.lua

```diff
@@ -0,0 +1,73 @@
+CFCAdverts = CFCAdverts or {}
+
+local advertCount = 0
+local curAdvert = 1
+local advertKeys = {}
+local timerName = CFCAdverts.HOOK_BASE .. "_SendAdvert"
```

Comment:
Could plop these onto `CFCAdverts` (maybe a subtable of it, even)

Reply:
Aside from curAdvert (which is still sensitive and can be broken if modified externally), all of those variables are merely for optimization, keeping track of info whenever it updates instead of having to run table.Count() or table.GetKeys() and such for every advert. Attaching them to the global table would merely give external access to something that should never be touched.

Reply:
Being able to debug values like that is useful. Technically, anyone could still find and modify these variables if they had the access. It's not straightforward at all, but it _is_ possible.

I think it's more useful than risky.

Reply:
Not quite sure how much useful info someone will be able to get out of for debugging if I de-localized those variables, but if you say so...